roles/confluent.test/molecule/custom-user-plaintext-rhel/molecule.yml

```diff
@@ -0,0 +1,127 @@
+        groups:
+          - schema_registry
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: kafka-rest1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: kafka-rest1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - kafka_rest
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: kafka-connect1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: kafka-connect1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - kafka_connect
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: ksql1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: ksql1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - ksql
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: control-center1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: control-center1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - control_center
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        ## Commenting below to avoid port collision on jenkins server
+        ## Uncomment to view c3 in browser at localhost:9021
+        # published_ports:
+        #   - "9021:9021"
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    provisioner:
+      name: ansible
+      config_options:
+        defaults:
+          hash_behaviour: merge
+      playbooks:
+        converge: ../../../../all.yml
+      inventory:
+        group_vars:
+          all:
+          zookeeper:
+            scenario_name: custom-user-plaintext-rhel
+          schema_registry:
+            schema_registry_user: sr-custom
+            schema_registry_group: sr-custom-group
+            appender_log_path: /kafka/logs/schemaregistry/
+          kafka_rest:
+            kafka_rest_user: rest-custom
+            kafka_rest_group: rest-custom-group
+            appender_log_path: /kafka/logs/kafkarest/
```

Comment:
can we add verify tasks that show the log dirs are configured the way they should be?